### PR TITLE
Add travis ci

### PR DESCRIPTION
**Pull request by gilgamezh**
- travis CI
- coveralls.io

### CODE REVIEW

**Comment by gilgamezh** [comment]
badges added to README

image

to merge test are required

image

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+python:
+      - "3.4"
+      - "3.5"
+      - "3.5-dev"
+      - "3.6"
+      - "3.6-dev"
+      - "nightly" # currently points to 3.7-dev
+install:
+    - "pip install -r requirements.txt"
```
Comment (Juanlu001) [inline .travis.yml]: Bear in mind that `pip install -e .[dev]` already does this. I prefer the shorter method but won't argue much about it, however we should probably be consistent with `CONTRIBUTING.md` and avoid duplicating the places where we specify the dependencies.

Comment (gilgamezh) [reply .travis.yml]: good catch! I'm removing it. 

And what about the requirements-test.txt?

**Comment by Juanlu001** [comment]
That's the `[dev]` part, which is in `extra_requires` in `setup.py`

**Comment by matiasg** [comment]
@gilgamezh  is the `travis.yml` correct?
I'm seeing this in travis site:
```==================================== ERRORS ====================================
__________________ ERROR collecting tests/test_quaternions.py __________________
ImportError while importing test module '/home/travis/build/satellogic/quaternions/tests/test_quaternions.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/test_quaternions.py:6: in <module>
    from quaternions import Quaternion, QuaternionError
E   ImportError: No module named 'quaternions'
!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!
```

**Comment by Juanlu001** [comment]
@matiasg where are you seeing that? Here all the status checks passed.

**Comment by matiasg** [comment]
@Juanlu001 , I was looking at the first commit in the PR. My bad, had to look at the last one.

**Comment by Juanlu001** [comment]
Sorry, wrong button :)

**Pull request merged**